packages/pluggableWidgets/color-picker-web/CHANGELOG.md

```diff
@@ -4,3 +4,8 @@ All notable changes to this widget will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [2.0.0] - 2022-01-11
```

Comment:
You should remove this for now, github actions automatically add this when we release it.